src/kubeobjects/secret.go

```diff
@@ -70,6 +94,49 @@ func (query SecretQuery) CreateOrUpdate(secret corev1.Secret) error {
 	return nil
 }
 
+func (query SecretQuery) CreateOrUpdateForNamespacesList(newSecret corev1.Secret, namespaces []corev1.Namespace) error {
+	secretList, err := query.GetAllFromNamespaces(newSecret.Name)
+	if err != nil {
+		return err
+	}
+
+	query.log.Info("creating/updating secret for multiple namespaces",
```

Comment:
```suggestion
	query.log.Info("reconciling secret for multiple namespaces",
```

Reply:
this log could possible be removed as the following looks a bit redundant
```
...
{"level":"info","ts":"2023-01-03T10:19:30.069Z","logger":"initgeneration","msg":"creating/updating secret for multiple namespaces","name":"dynatrace-dynakube-config","len(namespaces)":1 }
{"level":"info","ts":"2023-01-03T10:19:30.069Z","logger":"initgeneration","msg":"created/updated secret for multiple namespaces","name":"dynatrace-dynakube-config","creationCount":0,"updateCount":0}
...
```

Reply:
Would not remove it otherwise it would print "querying secret from all namespaces" as last msg before the secrets are created. If there is a greater amount of namespaces this can take a while, which could lead to the false assumption that the querying takes its time.